app/controllers/teams_controller.rb

```diff
@@ -36,9 +36,13 @@ def update
     end
   end
 
-  def destroy
-    @team.destroy
-    redirect_to teams_url, notice: 'Team was successfully destroyed.'
+  def delete_memberships
```

Comment:
Ez legyen inkább a default destroy metódus (elég csak átnevezni), illetve fent a `before_action :set_team`-hez add hozzá a destroy-t is.